src/interfaces/response_object.interface.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2018 Amsterdam University of Applied Sciences (AUAS)
+ *
+ * This software is distributed under the terms of the
+ * GNU General Public Licence version 3 (GPL) version 3,
+ * copied verbatim in the file "LICENSE"
+ */
+
+ /**
+  * Interface to standardize the response of a API object
+  */
+export interface ResponseObject {
```

Comment:
Create two separate response objects similar to how you have done that in the UI, one for item and one for items. Also use generics like you do in the UI.

Reply:
Try to think of good names, that are the same as in the UI, as I have commented in the ui PR. https://github.com/SoftwareForScience/jiskefet-ui/pull/54#discussion_r249942143